modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,541 @@
+= XDCR Conflict Logging
+:description: pass:q[XDCR detects and logs concurrent conflicts that occur in different clusters but on the same document version due to independent modifications by locally connected applications during Active-Active replication. The conflict logs are for your information only. The best practice in Active-Active systems is that application environments must be designed to avoid conflicts.]
+:page-toclevels: 3
+
+[abstract]
+{description}
+
+During XDCR Active-Active replication, a conflict can occur if different applications connected locally to two different clusters, independently modify the same version of the document at around the same time.
+These simultaneous modifications happen within the convergence window*, so the changes made in one document are not reflected in the history of the other.
+
+If conflict logging is enabled, XDCR logs these conflict events in a conflict log collection, specified by you, for your information.
+The conflict logging does not change conflict resolution.
+
+XDCR performs conflict resolution and replicates the documents whether conflicts are logged or not.
+Conflict logging is informational only, so that you can review the conflict log message documents and the copies of the documents that were in conflict, to help you design application systems that avoid conflicts.
+
+_*A convergence window is a period during which multiple actors, such as users, applications, or clusters, can modify the same document in an active-active topology, before the changes have fully converged across all clusters._
+
+== Adaptibility of XDCR Conflict Logging
+
+* Conflict logging is asynchronous and done on a best-effort basis, meaning, not all conflicts may be logged.
+
+* Conflicts are expected to be rare during XDCR replication.
+If there is an upsurge of conflicts, so that logging conflicts may cause replication issues, XDCR stops logging conflicts temporarily to prioritize replication.
+
+* If any error occurs, due to resource issues or a delay in writing to the conflict log collection, XDCR may pause or stop the logging process, report the error type for the pipeline, and continue with replication so that there is no High Availability issue.
+
+* xref:learn:clusters-and-availability/xdcr-conflict-resolution.adoc[Conflict Resolution] process remains unchanged.
+
```

Comment:
IMPORTANT:  Even if conflict logging fails, XDCR continues with conflict resolution to ensure High Availability.  Conflict resolution is not changed by conflict logging.  Conflict logging is informational only.